packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -75,3 +83,31 @@ const memoizedUpdateDisplayName = memoizeFnAll((participantId: string, participa
     return participant;
   }
 });
+
+/**
+ * @private
+ */
+export const isTeamsCallParticipants = (
+  participants: CommunicationIdentifier[]
+): participants is (PhoneNumberIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier)[] => {
+  for (const p of participants) {
```

Comment:
The one-liner lover in me is itching to say: `return participants.every(p => !isCommunicationUserIdentifier(p))`